src/main/views/cop-daily-cause-list.njk

```diff
@@ -0,0 +1,139 @@
+      <div class="govuk-details__text">
+        <p>{{ importantInformationP1 }}</p>
+        <p>{{ importantInformationP2 }}</p>
+        <p>{{ listData['venue']['venueContact']['venueEmail'] }} {{ listData['venue']['venueContact']['venueTelephone'] }}</p>
+        <p>{{ importantInformationP3 }}</p>
+        {% if bill %}
+          <P>{{ importantInformationP4 }}</P>
+          <P>{{ importantInformationP5 }}</P>
+          <p>{{ listData['venue']['venueContact']['venueEmail'] }} {{ listData['venue']['venueContact']['venueTelephone'] }}</p>
+          <P>{{ importantInformationP6 }}</P>
+        {% endif %}
+      </div>
+    </details>
+    {% for courtList in listData['courtLists'] %}
+      {% set courtHouse = courtList['courtHouse'] %}
+      <div class="site-header" tabindex="0" aria-label="Court Details">
+        <h1 class="govuk-heading-l site-address" tabindex="0">{{ courtHouse['courtHouseName'] }}</h1>
+      </div>
+      <div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
+      {% set sessionCount = 0 %}
+      {% for courtRoom in courtHouse['courtRoom'] %}
+        {% set hearingCount = 1 %}
+        {% set sessionCount = sessionCount + 1 %}
+        {% for session in courtRoom['session'] %}
+          <div class="govuk-accordion__section govuk-accordion__section--expanded">
+            <div class="govuk-accordion__section-header">
+              <h2 class="govuk-accordion__section-heading">
+            <span class="govuk-accordion__section-button" id="accordion-default-heading-{{ sessionCount }}">
+              {{ beforeHon }} {{ session['formattedJudiciary'] }}
+            </span>
+              </h2>
+            </div>
+
+            <div id="accordion-default-content-{{ sessionCount }}" class="govuk-accordion__section-content"
+                 aria-labelledby="accordion-default-heading-{{ sessionCount }}">
+              <div class="parent-box overflow-table">
+                <table class="govuk-table">
+                  <thead class="govuk-table__head">
+                  <tr class="govuk-table__row">
+                    <th scope="col" class="govuk-table__header">{{ startTime }}</th>
+                    <th scope="col" class="govuk-table__header">{{ caseRef }}</th>
+                    <th scope="col" class="govuk-table__header">{{ caseDetails }}</th>
+                    <th scope="col" class="govuk-table__header">{{ hearingType }}</th>
+                    <th scope="col" class="govuk-table__header">{{ timeEstimate }}</th>
+                    <th scope="col" class="govuk-table__header">{{ hearingChannel }}</th>
+                  </tr>
+                  </thead>
+                  <tbody class="govuk-table__body">
+                  {% for sitting in session['sittings'] %}
+                    {% set durationAsHours = 0 %}
+                    {% if sitting['durationAsHours'] !== 0 %}
```

Comment:
If this check here required? Inside the 'if' statement, there are already check that the value is '>1''or '==1' so it will not be zero

Reply:
Done :)